core/misc/tabledrag.js

```diff
@@ -1,5 +1,20 @@
 (function ($) {
 
+"use strict";
+
+/**
+ * Store the state of weight columns display for all tables.
+ *
+ * Default value is to hide weight columns.
+ *
+ * We are using JSON.parse here, because true/false get stored as strings.
+ * Otherwise showWeight would be either 'true' or 'false', and they would both
+ * be true in the toggle condition.
+ */
+var showWeight = JSON.parse(
```

Comment:
Any chance to avoid this global variable?

Reply:
Not sure @indigoxela 🤷🏼 ...my goal was to simply backport https://git.drupalcode.org/project/drupal/-/commit/148871a8c914c2db9c28ceac3ebf9763a0e83e7e

If there's a way to achieve that, then please feel free to suggest via PR against mine, or a completely new PR. Would be great to also repay our Drupal brethren by contributing the same back to Drupal.